sample-apps/celery_sample/blog/tasks.py

```diff
@@ -0,0 +1,14 @@
+import time
+
+# from celery_sample.celery import app as celery_app
+# from celery_sample.celery import CeleryEventsHandler
+from celery import shared_task
+
+
+@shared_task(bind=True)
```

Comment:
The sample app should have some way to trigger this task without running a python shell. For my testing, I added it in the blog index view so a task is enqueued each time I loaded localhost:5000/.

Reply:
True, but resolving the reporting and debugging the task's error tracing is easier from the python shell. So that was my focus till now, to resolve the metrics reporting